endroid/data-sanitize/1.3/config/packages/dev/endroid_data_sanitize.yaml

```diff
@@ -0,0 +1,14 @@
+endroid_data_sanitize:
+    entities:
+        project:
+            class: Endroid\DataSanitize\Bundle\DataSanitizeDemoBundle\Entity\Project
```

Comment:
I don't think that the demo bundle should be configured when creating a new project

Reply:
That's true, ideally not